project/src/components/card-property-not-logged/card-property-not-logged.tsx

```diff
@@ -28,36 +60,23 @@ function CardPropertyNotLogged(): JSX.Element {
         <section className="property">
           <div className="property__gallery-container container">
             <div className="property__gallery">
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/room.jpg" alt="Room"/>
-              </div>
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/apartment-01.jpg" alt="Room"/>
-              </div>
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/apartment-02.jpg" alt="Room"/>
-              </div>
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/apartment-03.jpg" alt="Room"/>
-              </div>
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/studio-01.jpg" alt="Room"/>
-              </div>
-              <div className="property__image-wrapper">
-                <img className="property__image" src="img/apartment-01.jpg" alt="Room"/>
-              </div>
+              {images.map((image) => (
+                <div className="property__image-wrapper" key={id}>
+                  <img className="property__image" src={`${image}`} alt=""/>
+                </div>),
+              )}
             </div>
           </div>
           <div className="property__container container">
             <div className="property__wrapper">
-              <div className="property__mark">
-                <span>Premium</span>
-              </div>
+              {isPremium ?
+                <div className="property__mark"><span>Premium</span>
+                </div> : ''}
               <div className="property__name-wrapper">
                 <h1 className="property__name">
-                  Beautiful &amp; luxurious studio at great location
+                  {title}
                 </h1>
-                <button className="property__bookmark-button button" type="button">
+                <button className={isFavorite ? 'property__bookmark-button property__bookmark-button--active button' : 'property__bookmark-button  button'} type="button">
```

Comment:
Тут можно избавиться от дублирования классов, если использовать обратные кавычки

Смотри, как:

```suggestion
                <button className={`property__bookmark-button button ${isFavorite ? property__bookmark-button--active : null }`} type="button">
```

Тут мы просто оставили два класса, которые есть в любом случае - и делам проверку только на один